src/_data/side-nav.yml

```diff
@@ -49,15 +49,17 @@
   - title: "Dart Testing"
     permalink: /guides/testing
 
-- title: "Resources"
+- title: Resources
   children:
-  - title: "Codelabs"
+  - title: Install
```

Comment:
I expected Install to be bold when I was on a page under it.

Is the only way to get that behavior to add the installation subpages to the sidenav? Is there any reason not to add the subpages there?

Reply:
The subnav code would need to be fixed. I'd prefer addressing that separately (I'm noticing that this incorrect behavior is already there for the Tools pages).

I've created https://github.com/dart-lang/site-www/issues/403 to track this.